### PR TITLE
Update to disable CI triggers

### PR DESCRIPTION
**Pull request by weshaggard**

### CODE REVIEW

**Comment by weshaggard** [comment]
This updates pulls in these changes to pipeline generation tool https://github.com/Azure/azure-sdk-tools/pull/560

File: eng/pipelines/pipeline-generation.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ trigger: none
 
 variables:
   skipComponentGovernanceDetection: true
-  PipelineGeneratorToolVersion: 1.0.2-dev.20200423.1
+  PipelineGeneratorToolVersion: 1.0.2-dev.20200502.1
```
Comment (JimSuplizio) [inline eng/pipelines/pipeline-generation.yml:7]: @weshaggard Crazy question but where is this version coming from?

Comment (weshaggard) [reply eng/pipelines/pipeline-generation.yml:7]: That is the default build numbers coming from our tools build pipeline. Similar to our daily dev versions numbers.

**Pull request merged**